package.json

```diff
@@ -12,12 +12,18 @@
     "pretest": "lerna run pretest",
     "test": "node scripts/test.js",
     "posttest": "lerna run posttest",
-    "test:watch": "node scripts/test.js --watch",
-    "lint": "tsc -b && yarn prettier:check && eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
-    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix; prettier --ignore-path .eslintignore --write '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "prettier:check": "prettier --ignore-path .eslintignore --check '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "depcheck": "lerna exec --stream 'yarn depcheck --ignores=\"@app/config,@app/client,tslib,webpack,babel-plugin-import,source-map-support,@graphql-codegen/*,graphql-toolkit,net,tls,dayjs\" --ignore-dirs=\".next\"'",
-    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently --kill-others --names 'TSC,WATCH,RUN,TEST' --prefix '({name})' --prefix-colors 'yellow.bold,yellow.bold,cyan.bold,greenBright.bold' 'tsc -b --watch --preserveWatchOutput --assumeChangesOnlyAffectDirectDependencies' 'lerna run --parallel watch' 'lerna run --parallel dev' 'sleep 10; yarn test:watch'",
+    "test:watch": "run-script-os",
+    "test:watch:darwin:linux": "sleep 10 && yarn test:watch:default",
+    "test:watch:default": "node scripts/test.js --watch",
+    "lint": "tsc -b && yarn prettier:check && yarn eslint:check",
+    "lint:fix": "run-script-os",
+    "lint:fix:windows": "yarn eslint:check --fix & yarn prettier:fix",
+    "lint:fix:default": "yarn eslint:check --fix; yarn prettier:fix",
+    "eslint:check": "eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
+    "prettier:check": "prettier --ignore-path .eslintignore --check \"**/*.{js,jsx,ts,tsx,graphql,md}\"",
+    "prettier:fix": "prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,ts,tsx,graphql,md}\"",
+    "depcheck": "lerna exec --stream \"yarn depcheck --ignores=\"@app/config,@app/client,tslib,webpack,babel-plugin-import,source-map-support,@graphql-codegen/*,graphql-toolkit,net,tls,dayjs\" --ignore-dirs=\".next\"\"",
+    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently --kill-others --names \"TSC,WATCH,RUN,TEST\" -p \"({name})\" -c \"yellow.bold,yellow.bold,cyan.bold,greenBright.bold\" \"tsc -b --watch --preserveWatchOutput\" \"lerna run --parallel watch\" \"lerna run --parallel dev\" \"yarn test:watch\"",
```

Comment:
Do we need to use the short aliases for Windows, or can we stick with the longer ones that are more legible/obvious?

Reply:
I am not entirely sure what you mean here. I split up the prettier fixes as Windows does not support running two commands using ';', so we have another `lint:fix` command that changes the syntax based on the OS. 

Or is that not what you mean?

Reply:
Oh sorry! You changed `concurrently` to use `-p` and `-c` rather than `--prefix` and `--prefix-colors`. I prefer long flags because they're more obvious to people unfamiliar with the tools; please change them back unless there's a good reason not to :+1: